core/src/main/gamelogic/Referee.kt

```diff
@@ -0,0 +1,160 @@
+package gamelogic
+
+import gamelogic.map.PoliticalMap
+import gamelogic.occupations.CountryOccupations
+import Country
+import Player
+import PositiveInt
+import gamelogic.combat.Attacker
+import gamelogic.combat.Conqueror
+import gamelogic.combat.resolver.CombatDiceRoller
+import gamelogic.combat.resolver.DiceRollingCombatResolver
+import gamelogic.dice.RandomDie
+import gamelogic.situations.classicCombat.ClassicCombatDiceAmountCalculator
+
+
+//TODO: En lugar de crear esta clase lo que hay que hacer es sacar del ataquer la
+//      responsabilidad de mover las fichas después de atacar. Peeero, no solo hay
+//      que hacer refactor del package combat, sino también de los tests!!! Asi que
+//      queda para otro backlog item.
+class SkipRegroup : Conqueror {
+    override fun armiesToMove(remainingAttackingArmies: PositiveInt): PositiveInt {
+        return PositiveInt(0)
+    }
+
+}
+
+class CountryReinforcement(val country:Country, val n: PositiveInt) {
```

Comment:
:scream: una variable de una sola letra. Eso no lo podemos permitir!
```suggestion
class CountryReinforcement(val country:Country, val armies: PositiveInt) {
```

Reply:
Che, ahora que veo estas clases así hechas que hacen cada comando tipo de country reinforcement y de Regrouping y esas cosas, me hace acordar creo que se llama al Command pattern que cada acción del usuario es un command, y entonces tenés una lista de Commands que le llamás apply a cada una y el referee como que se convierte solo en un llamador de Commands. Esto te deja después guardar fácil los commands que ocurrieron en un juego para después podes ver una repe re fácil, o también deshacer una acción que realizaste y te equivocaste, como por ejemplo reagrupar algo que no querías. Quizá sería copado hacer eso